fastlane/lib/fastlane/actions/slather.rb

```diff
@@ -52,7 +53,22 @@ def self.has_config_file
         File.file?('.slather.yml')
       end
 
+      def self.slather_version(params)
+        command = []
```

Comment:
Instead of shelling out to receive the version number, can't we do something like
```ruby
require 'slather'
Slather::VERSION
```